src/merkle_tree.rs

```diff
@@ -207,9 +215,9 @@ where
         }
     }
 
-    pub fn build_merkle_proof(leaves: &[T], indices: &[u32]) -> Option<MerkleProof<T, M>> {
+    pub fn build_merkle_proof(leaves: &[T], indexes: &[u32]) -> Option<MerkleProof<T, M>> {
```

Comment:
Please revert the renaming and keep the name consistent with `proof.indices`.

Reply:
The argument here is actually the indexes of txs in block. `proof.indices` is the indexes of node in merkle tree. They are different, and cause confusion.